lib/agent.js

```diff
@@ -243,7 +243,7 @@ Agent.prototype.captureError = function (err, opts, cb) {
   var agent = this
   var trans = this.currentTransaction
   var timestamp = new Date().toISOString()
-  var id = uuid.v4()
+  var id = crypto.randomBytes(128 / 8).toString('hex')
```

Comment:
Actually, it seems the spec implies this should be 64 bits. https://github.com/elastic/apm-server/blob/v2/docs/spec/errors/v2_error.json#L12